journal/pom.xml

```diff
@@ -142,7 +142,17 @@
           </usedDependencies>
         </configuration>
       </plugin>
-
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Why is this needed?

Reply:
To make LogCorrupter (which is in the journal tests) available for the qa tests